docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Persisted Queries
+---
+
+Apollo supports features called "Persisted Queries" and "Automatic Persisted Queries" (APQs), to avoid sending the same large query documents over and over, and to allow safelisting of queries.
+
+Each query or mutation is identified by the SHA256 hash of its contents that can be sent to the server in lieu of sending a full query document. For APQs specifically, if the hash can't be found by the server, it sends back an error indicating that it needs the full query. If it receives this specific error, Apollo iOS will automatically retry the operation with the full query document without you having to do anything.
```

Comment:
nit: Do we want to say "each operation" since operations now include subscriptions as well?

Also PQs does allow for user-customizable identifiers that are _not_ the SHA256 hash of the document contents, but we could specify that in a note rather than avoiding the nice simplicity you have here. Maybe a block comment below this could call it out?

Reply:
So saying instead of "Each query or mutation" say "Each operation"?

Also we don't currently have support for supplying custom operation identifiers implemented in iOS.

Reply:
Agree with replacing "Each query or mutation" with "Each operation". I'm trying to consistently use "operation" instead of "query" in the [main PQ docs](https://github.com/apollographql/docs/pull/496).